Objects/typevarobject.c

```diff
@@ -168,7 +168,7 @@ constevaluator_call(PyObject *self, PyObject *args, PyObject *kwargs)
         return NULL;
     }
     PyObject *value = ((constevaluatorobject *)self)->value;
-    if (format == 3) { // STRING
+    if (format == 4) { // STRING
```

Comment:
The formats should be defined in C too.  Macros are fine.  I'm expecting `FORMAT_STRING` here, optionally with some variant of `Py_` in front.  Worst case, `Py_ANNOTATE_FORMAT_STRING`.

Reply:
I added a C enum for these.